src/routes/events/event.py

```diff
@@ -32,3 +35,41 @@ def latest_event():
     data = connection()
     latest = data[-1]
     return latest
+
+
+@event_bp.route("/announcement", methods=["POST"])
+def make_announcement():
+    API_KEY = os.getenv("API_KEY")
+    WEBHOOK_URL = os.getenv("WEBHOOK_URL")
+
+    api_key = request.args.get("api_key")
+
+    if not (api_key == API_KEY) or not API_KEY:
+        return "Unauthorized: Invalid API Key", 401
+
+    if not WEBHOOK_URL:
+        return (
+            "Internal Error: No Webhook URL is configured in environment variables",
+            500,
+        )
+
+    data = connection()
+    post_data = request.json
+
+    if not post_data or not post_data.get("event_id"):
```

Comment:
If no event ID was provided in POST request, then take event_id from the latest event.
Why? Event ID data is auto-generated by Django and added to the database. When a new event is created, this endpoint is called without an event id.

Reply:
Handled in 3524cc2

Reply:
![image](https://user-images.githubusercontent.com/43880032/145668350-74bc923a-c757-4294-9d61-dc0153153e0f.png)